s3/sc-s3-synapse-ra.yaml

```diff
@@ -73,7 +72,11 @@ Resources:
             Sid: "WriteAccess"
             Effect: "Allow"
             Principal:
-              AWS: !Ref 'S3UserARNs' #sc-requestor write access
+              AWS: !Split
+                - ","
+                - !Sub
+                  - "${list},arn:aws:iam::325565585839:root" 
+                  - list: !Join [",",!Ref "S3UserARNs"]
```

Comment:
@ahayden I haven't ever seen this syntax before `list: ` -- do you have a reference for it? I can't find anything.

Reply:
wacko! https://bl.ocks.org/magnetikonline/a6cfc522a1e9f876b75962f5f553c8e5

Reply:
I saw it in this gist... https://gist.github.com/magnetikonline/a6cfc522a1e9f876b75962f5f553c8e5